checking/integration/support/steps.go

```diff
@@ -0,0 +1,174 @@
+package support
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/optimizely/go-sdk/optimizely/event"
+
+	"github.com/DATA-DOG/godog/gherkin"
+	"github.com/optimizely/go-sdk/checking/integration/optimizely/datamodels"
+	"github.com/optimizely/go-sdk/checking/integration/optimizely/eventdispatcher"
+	"github.com/optimizely/go-sdk/optimizely/entities"
+	"gopkg.in/yaml.v3"
+)
+
+// Context holds both request and response for a scenario
+type Context struct {
+	requestParams  datamodels.RequestParams
```

Comment:
Let's not call these request and response because they have "web" connotations. Instead we can just call them `apiOptions` and `apiResponse`